scapy/contrib/macsec.uts

```diff
@@ -0,0 +1,267 @@
+# MACsec unit tests
+# run with:
+#   test/run_tests  -P "load_contrib('macsec')" -t scapy/contrib/macsec.uts -F
+
++ MACsec
+~ crypto not_pypy
+
+= MACsec - basic encap - encrypted
+sa = MACsecSA(sci='\x52\x54\x00\x13\x01\x56\x00\x01', an=0, pn=100, key=b'aaaaaaaaaaaaaaaa', icvlen=16, encrypt=1, send_sci=1)
```

Comment:
Please update all the tests using: 
- bytes marks
- raw instead of str to build packets